integration_tests/specs/dom/elements/input.ts

```diff
@@ -1,4 +1,4 @@
-describe('Tags input', () => {
+xdescribe('Tags input', () => {
```

Comment:
Why skip this test?

Reply:
The input and textarea support are migrate to another PR.